grails-app/controllers/org/pih/warehouse/core/ErrorsController.groovy

```diff
@@ -97,8 +97,8 @@ class ErrorsController {
         if (RequestUtil.isAjax(request)) {
             response.status = 400
             BeanPropertyBindingResult errors = request?.exception?.cause?.errors
-            def errorMessages = errors.allErrors.collect {
-                return messageService.getMessage(it.codes[0], it.arguments, (it.defaultMessage ?: it.codes[0]), localizationService.getCurrentLocale())
+            List<String> errorMessages = errors.allErrors.collect {
+                return g.message(error: it, locale: localizationService.currentLocale)
```

Comment:
I think we discussed this before and I don't think we should be changing this right now unless we know for sure this is not going to break something.

Reply:
If you're running into an issue please raise it in the ticket and mark the ticket with the tech huddle label so we can discuss and try to resolve it.

Reply:
Oops, just saw the extensive discussion above. I'll review that discussion and provide some feedback tomorrow.

Reply:
> so maybe either @jmiranda or @awalkowiak might know, but feels like calling it.codes[0] makes no sense, since those "codes" are something like this

Just to elaborate on this ... it did make sense at a time when we weren't translating error messages because all of those codes should point to the same localization message. So we just needed any one of the codes to display the error message properly. Now that we're translating messages, we need to be more exact with which error code we use.